pkg/storage/export/virt-exportserver/exportserver.go

```diff
@@ -205,6 +260,136 @@ func NewExportServer(config ExportServerConfig) service.Service {
 	return es
 }
 
+var getExpandedVM = func() *virtv1.VirtualMachine {
+	f, err := os.Open(vmManifestPath)
+	if err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+	defer f.Close()
+	fileinfo, err := f.Stat()
+	if err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+	buf := make([]byte, fileinfo.Size())
+	_, err = f.Read(buf)
+	if err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+
+	vm := &virtv1.VirtualMachine{}
+	if err := json.Unmarshal(buf, vm); err != nil {
+		log.Log.Reason(err).Info("Unable to load VM manifest data")
+		return nil
+	}
+	return vm
+}
+
+var getInternalBasePath = func() (string, error) {
+	data, err := os.ReadFile(internalLinkPath)
+	if err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
+
+var getExportName = func() (string, error) {
+	data, err := os.ReadFile(exportNamePath)
+	if err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
+
+var getExternalBasePath = func() (string, error) {
+	data, err := os.ReadFile(externalLinkPath)
+	if err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
+
+func GetTypeMetaString(gvk schema.GroupVersionKind) string {
+	return fmt.Sprintf("apiVersion: %s\nkind: %s\n", gvk.GroupVersion().String(), gvk.Kind)
+}
+
+var getCAConfigMap = func(name string) (*corev1.ConfigMap, error) {
+	f, err := os.Open(name)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	fileinfo, err := f.Stat()
+	if err != nil {
+		return nil, err
+	}
+	buf := make([]byte, fileinfo.Size())
+	_, err = f.Read(buf)
+	if err != nil {
+		return nil, err
+	}
+
+	cm := &corev1.ConfigMap{}
+	if err := json.Unmarshal(buf, cm); err != nil {
+		return nil, err
+	}
+	return cm, nil
+}
+
+var getCdiHeaderSecret = func(token, name string) *corev1.Secret {
+	data := make(map[string]string)
+
+	data["token"] = fmt.Sprintf("x-kubevirt-export-token:%s", token)
```

Comment:
probably doesn't matter but may want to add a space before %s:  `"x-kubevirt-export-token: %s"`

Reply:
it actually does matter, if you add the space it won't work.

Reply:
huh, that is confusing and may point to a problem somewhere.  For example, `curl  --insecure https://localhost:18443/internal/manifests/secret -H "x-kubevirt-export-token: 5kvJ2j4KPl"`  works fine

Reply:
okay we should probably create a github issue for that